src/docs/patterns/user-auth.mdx

```diff
@@ -0,0 +1,324 @@
+
+-   [Login with Amazon](https://developer.amazon.com/docs/login-with-amazon/brand.html)
+-   [Google Sign-in](https://developers.google.com/identity/sign-in/web/sign-in)
+-   [Sign in with Apple](https://developer.apple.com/design/human-interface-guidelines/sign-in-with-apple/overview/introduction/)
+-   [Facebook Login](http://developers.facebook.com/docs/facebook-login/)
+-   [Sign in with Microsoft](https://developer.microsoft.com/en-us/identity/add-sign-in-with-microsoft)
+
+## Forgot / Reset Password
+
+**Forget password**
+
+The login screen for an application should also present users with a way to recover or reset their password if they have lost or forgotten it. This is typically presented as a hyperlink below the credential fields. Clicking the link will present users with a field to enter their e-mail address. They will receive an email with a hyperlink to reset their password. This workflow involves the following steps:
+
+-   Enter email address
+-   Success: reset instructions sent
+-   Receive an email for resetting the password
+-   Create a new password
+-   Success: password reset
+
+<ImageGrid
+    images={[ForgotPasswordEnterEmail, ForgotPasswordConfirmSent, ForgotPasswordNewPassword, ForgotPasswordWelcome]}
+    caption={['1: Enter email address', '2: Success: reset instructions sent', '3: Create a new assword', '4: Success: password reset']}
+/>
+<ImageGrid images={[ResetPasswordDesktop]} caption={'A reset-password screen example on desktop.'} regularWidth />
+
+**Change password**
+
+When the users are logged in, they have the option to change their password by going through the change password workflow, typically triggered within the user settings. This workflow includes the following steps:
+
+-   Change password
+-   Success: password changed
+
+<React.Fragment>
+    <ImageGrid
+        images={[ChangePassword, ChangePasswordWelcome]}
+        caption={['1: Change password', '2: Success: password changed']}
+    />
+    <ImageGrid images={[ChangePasswordDesktop]} caption={'A change-password screen example on desktop.'} regularWidth />
+    <Divider />
+</React.Fragment>
+
+# Behavior
+
+## Email Communications
+
+<!-- This might deserve its own pattern. We already have something in the backlog to design for an email template. -->
+
+Email communications are frequently used throughout the user authentication workflow, as they are the primary way a user will get notified without an account access. Users receive emails for invitations, verification, confirmation codes, etc.
+
+Emails are also sent to confirm security-related changes to a user's account (such as changing a password, updating a profile, etc.) You should never allow users to unsubscribe from security-related email notifications.
+
+<ImageGrid
+    images={[EmailCommunicationInvitation, EmailCommunicationVerification]}
+    caption={['Two examples of email communications', '1: Invitation to join', '2: Verifying email address']}
+/>
+
+## Success Screen
```

Comment:
Does this belong in Behaviors? Or should it be somewhere further up?

Reply:
I understand what makes you say so. I put it there just because it feels cleaner.

Reply:
Is it short enough to maybe put it in a 
> callout ?